packages/react/src/MinimalFooter/MinimalFooter.features.stories.tsx

```diff
@@ -61,6 +61,21 @@ export const MaximumLinks = () => (
 )
 MaximumLinks.storyName = 'Maximum Links (5)'
 
+export const ButtonLinks = () => (
+  <MinimalFooter socialLinks={false}>
+    <MinimalFooter.Link as="button" onClick={() => alert('You have clicked Button one')}>
+      Button one
+    </MinimalFooter.Link>
+    <MinimalFooter.Link as="button" onClick={() => alert('You have clicked Button two')}>
+      Button two
+    </MinimalFooter.Link>
+    <MinimalFooter.Link as="button" onClick={() => alert('You have clicked Button three')}>
+      Button three
+    </MinimalFooter.Link>
+  </MinimalFooter>
+)
+MaximumLinks.storyName = 'Button Links'
```

Comment:
This is changing the wrong story, and is presenting like this in Storybook..

<img width="206" alt="Screenshot 2023-05-13 at 16 59 45" src="https://github.com/primer/brand/assets/13340707/2c4dc666-d999-429e-84b9-588c11b1ac19">

Recommend it's removed altogether as it's not any different to the default name.

```suggestion
```

Reply:
This has been completed in this [SHA](https://github.com/primer/brand/pull/272/commits/92097b009631322bf5496f5604e08b9e6c5716f5)